test/testIndependentSet.cpp

```diff
@@ -0,0 +1,153 @@
+#include <gtest/gtest.h>
```

Comment:
The examples are fine - but I have a small question: why did you decide against mirroring the outline of other tests? Do you think it's any better?

Please let me know, I still haven't decided on this, and it'd be great to have some input, as gtest is a little unwieldy.

Reply:
OK, let it be for the time being, I'll refactor it on the side